src/index.less

```diff
@@ -1,476 +1,338 @@
-                margin: 0;
-                padding: 0 10px;
-              }
-            }
-
-            &.active {
-              > .item-content > .menu-item {
-                background-color: @SIDEBAR_MENU_ACTIVE_ITEM_BG_COLOR;
-                color: @SIDEBAR_MENU_TEXT_ACTIVE_COLOR;
-                font-weight: bold;
-              }
-            }
-
-            .menu-level-3 {
-              background-color: @SIDEBAR_MENU_BG_COLOR_LVL3;
-              padding: 6px 0 10px 56px;
-
-              > li.item-container {
-                > .item-content > .menu-item {
-                  padding: 0 10px;
-
-                  .item-icon i {
-                    font-size: @SIDEBAR_MENU_LVL3_ICON_FONT_SIZE;
-                    margin-bottom: 5px;
-                  }
-
-                  .item-arrow.no-sub {
-                    width: 10px;
-                  }
-                }
-
-                &.active {
-                  > .item-content > .menu-item {
-                    background-color: @SIDEBAR_MENU_ACTIVE_ITEM_BG_COLOR;
-                    color: @SIDEBAR_MENU_TEXT_ACTIVE_COLOR;
-                    font-weight: bold;
-                  }
-                }
-
-                .menu-level-4 {
-                  > li.item-container {
-                    &.active {
-                      > .item-content > .menu-item {
-                        background-color: @SIDEBAR_MENU_ACTIVE_ITEM_BG_COLOR;
-                        color: @SIDEBAR_MENU_TEXT_ACTIVE_COLOR;
-                        font-weight: bold;
-                      }
-                    }
-                  }
-                }
-              }
-            }
-
-            &.open {
-              > .item-content {
-                > .group-content-wrapper > div > div > .group-toggle-content > .menu-view-all {
-                  background-color: @SIDEBAR_MENU_BG_COLOR_LVL3;
-                  padding: 0 20px 0 10px;
-
-                  > .menu-view-all-inner {
-                    padding-left: 40px;
-                  }
-                }
-
-                .menu-level-3 {
-                  .item-container {
-                    position: relative;
-                    overflow: hidden;
-
-                    &::before {
-                      content: " ";
-                      background-color: @SIDEBAR_MENU_LVL2_ACTIVE_BORDER_COLOR;
-                      width: 1px;
-                      height: 100%;
-                      display: inherit;
-                      position: absolute;
-                      left: 0;
-                    }
-                  }
-                }
-              }
-            }
-          }
-        }
+      // Sub menu items
+      .sidebar-menu-list .menu-item {
+        padding-left: @sidebar-sub-menu-padding-left;
       }
     }
+
+    &.menu-level-3 {
+      background: @sidebar-background-color_level-3;
+    }
+
+    &.menu-level-4 {
+      background: @sidebar-background-color_level-4;
+    }
+  }
+
+  .ms-Icon {
```

Comment:
Can we please avoid using this className as a selector?
It come from a third-party library which is not defined as a peer for this package.

It's imported only from the host applications.

Reply:
Put back in its previous place